### ISSUE TITLE
PY_SSIZE_T_CLEAN warning with python 3.8

### ISSUE THREAD

**Issue by bsergean** [ilanschnell/bsdiff4#11]
I am getting this warning, does it ring a bell ?

```
  /projects/pix/src/pix/minibsdiff.py:197: DeprecationWarning: PY_SSIZE_T_CLEAN will be required for '#' formats
    patchfile_data = bsdiff4.core.patch(oldfile_data, newlen,
```

**Comment by ilanschnell** [comment]
Thanks for reporting this issue!  I cannot reproduce this myself, but after looking into it some more, it seems like defining PY_SSIZE_T_CLEAN should fix the problem.  I have done this in 35a390c67bb5c07b01b3d7b271e354da7fb36952

I'm closing this now, but please reopen this issue if the problem should persist.  This fix will be part of bsdiff4 1.1.7, which I will release very soon.

**Comment by bsergean** [comment]
I tried installing latest bsdiff4 on my mac with python3.6 and 3.8, and I couldn't see this warning anymore, so your last commit must have fixed it.
